src/vmm/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "vmm"
+version = "0.1.0"
+authors = ["rust-vmm AWS maintainers <rust-vmm-maintainers@amazon.com>"]
+edition = "2018"
+
+[dependencies]
+event-manager = "0.2.0"
+kvm-bindings = { version = "0.3.0", features = ["fam-wrappers"] }
+kvm-ioctls = "0.5.0"
+linux-loader = "0.2.0"
+vm-device = { git = "https://github.com/aghecenco/vm-device.git", branch = "send" }
```

Comment:
This is temporary - will point to upstream `vm-device` once https://github.com/rust-vmm/vm-device/pull/31 is merged, or some other way to ensure thread safe bus accesses makes its way there.